File: Vachagan/C/caesar.c
#include <stdio.h>
#include <cs50.h>
#include <string.h>
#include <ctype.h>

int main(void){
    int key = get_int("Enter key : ")%26;
    string word = get_string("Enter word : ");
    
    for (int i = 0; i < strlen(word); i++) {
        if(isalpha(word[i])){
            int ascii = (word[i] < 97) ? 65 : 97;
            word[i] = (char) ((word[i] + key - ascii) % 26 + ascii);
        }
        printf("%c", word[i]);
    }

    printf("\n");

}

File: Garnik/CS50/Week6/Ceasar/caesar.py
from cs50 import get_string
from sys import argv

if len(argv) != 2:
    print("Usage: python [file] [key]")
    exit(1)

word = get_string("Please enter a word: ")
x = argv[1]
print("ciphertext: ", end="")

for i in word:
    if i.isalpha():
        i = chr(ord(i) + (int(x) % 26))
        if (not i.islower()) and (not i.isupper()):
            i = chr(ord(i)-26)

    print(i, end="")

print()